### DISCUSSION TITLE
[FeatureRequest] Variable Query: label_vaules() supports metrics operator operations

### DISCUSSION THREAD

**Discussion by TomorrowTown** [grafana/grafana#61364] [Dashboards, Panels & Visualizations]
Suppose I have an metric _metric1_, which has two labels: _label1_ and _label2_, the correct syntax to get _label2_ when _label1_="a" is:
**label_values(metric1(label1="a"), lable2)**

But this does not support further filtering of the results of metric1, for example:
**label_values(metric1(label1="a") == 20, lable2)**, or
**label_values(metric1(label1="a") < 10, lable2)**.
This will result in a syntax error.

This would be a very useful feature, I'm looking forward to seeing this in grafana, I wonder if the grafana team has any plans to develop this feature?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424).  As such, I am closing this ticket.

I feel like this is a feature request for the query language of a datasource, and the feature request would be best served for that datasource. If you let me know what you're querying, I can help point you in the right direction.
